File: Data_Preprocessing/data_preprocessing_template.py
# Data Preprocessing

# Importing the libraries
import numpy as np
import matplotlib.pyplot as plt
import pandas as pd

# Import the data set
dataset = pd.read_csv('Data.csv')
x = dataset.iloc[:, :-1].values
y = dataset.iloc[:, 3].values

# Split the dataset into training/testing sets
from sklearn.cross_validation import train_test_split
x_train, x_test, y_train, y_test = train_test_split(x, y, test_size = 0.2, random_state = 0)

# Feature Scaling
"""from sklearn.preprocessing import StandardScaler
sc_x = StandardScaler()
x_train = sc_x.fit_transform(x_train)
x_test = sc_x.transform(x_test)"""

File: Data_Preprocessing/data_preprocessing_template.R
# Data Preprocessing

# Import Data
dataset <- read.csv('Data.csv')
#dataset <- dataset[, 2:3]

# Split dataset into training/testing sets
library(caTools)
set.seed(123)
split <- sample.split(dataset$Purchased, SplitRatio = 0.8)
trainingset <- subset(dataset, split == TRUE)
testingset <- subset(dataset, split == FALSE)

# Feature Scaling
# trainingset[, 2:3] <- scale(trainingset[, 2:3])
# testingset[, 2:3] <- scale(testingset[, 2:3])
